### PR TITLE
Amos

### PR DESCRIPTION
**Pull request by davidamosprincipal**

### CODE REVIEW

File: branchExercise.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
+line 1 branch amos commit 1
```
Comment (gartee-john-PFG) [inline branchExercise.txt:1]: looks good

**Review by gartee-john-PFG — APPROVED** [review]
good to go

**Review by m043568 — COMMENTED** [review]
David - this doesn't look right.  Please correct this file.

File: branchExercise.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+line 2 branch amos commit 1
```
Comment (m043568) [inline branchExercise.txt:2]: Please fix this!

**Review by m043568 — CHANGES_REQUESTED** [review]

File: branchExercise.txt (hunk 2 above)
Comment (davidamosprincipal) [reply branchExercise.txt:2]: looks ok to me

**Review by m043568 — APPROVED** [review]

**Pull request merged**